### PR TITLE
Support max_time_ms in find()

### LINKED ISSUE #500: Support for max_time_ms in find()
I'm getting `find() got an unexpected keyword argument 'max_time_ms'` and I think adding `max_time_ms=None` to `def find()` would solve it easily as a first step, even without an actual implementation yet.

### PR DESCRIPTION
**Pull request by orik123**
Fixes #500

### CODE REVIEW

**Comment by pcorpet** [comment]
Thanks for the suggestion.

Please add:
* [ ] a test that would fail before your change
* [ ] a check for the type of `max_time_ms` if it's not `None`
* [ ] a comment on why we don't need an implementation (probably along the lines of the one in `Cursor.max_time_ms` method.

File: mongomock/collection.py [hunk 1]
```diff
@@ -907,12 +907,14 @@ def _discard_operators(self, doc):
...
-             batch_size=0, manipulate=True, collation=None, session=None):
+             batch_size=0, manipulate=True, collation=None, session=None,
+             max_time_ms=None):
         spec = filter
         if spec is None:
             spec = {}
         validate_is_mapping('filter', spec)
-        return Cursor(self, spec, sort, projection, skip, limit, collation=collation)
+        return Cursor(self, spec, sort, projection, skip, limit,
+                      collation=collation).max_time_ms(max_time_ms)
```
Comment (pcorpet) [inline mongomock/collection.py:917]: Smart :-)

**Review by pcorpet — APPROVED** [review]
Thanks for the changes.

**Comment by orik123** [comment]
Sure, my pleasure! Any idea when will be the next release?

**Comment by pcorpet** [comment]
Erf, I just did one yesterday…
I'll wait at least a week to see if there are bugs in this one and then publish a small fix with your change.

**Comment by orik123** [comment]
Great, thank you!

**Comment by orik123** [comment]
Thank you!

**Pull request merged**